docs/changelog.md

```diff
@@ -2,6 +2,8 @@
 
 ## 0.4.1 (In Progress)
 
+* **Breaking**
+    * Dusty now uses NFS instead of rsync to get repository code from your host Mac to running containers. The `sync` commands no longer exist, and `restart` no longer takes a `--no-sync` flag.
```

Comment:
This should be three separate bullet points. Also hype NFS more, mention bidirectional sync